docs/game-mechanics/behaviors/basic-attack.rst

```diff
@@ -1,39 +1,87 @@
-Basic Attack (1)
-================
+`Basic Attack (1) <https://github.com/DarkflameUniverse/DarkflameServer/blob/main/dGame/dBehaviors/BasicAttackBehavior.cpp>`_
+=============================================================================================================================
 
-This behavior is used to deal damage to the current target.
+This behavior is used to deal damage to a target.
 
-Parameters
-----------
+`Parameters <https://github.com/DarkflameUniverse/DarkflameServer/blob/main/dGame/dBehaviors/BasicAttackBehavior.cpp#L142>`_
+----------------------------------------------------------------------------------------------------------------------------
 
 .. list-table ::
-   :widths: 15
+   :widths: 15 30
    :header-rows: 1
 
    * - Name
+     - Description
    * - dir_angle_xz
+     - The direction to modify the attackers angle by.
    * - dir_angle_y
+     - The direction to modify the attackers angle by.
    * - dir_force
+     - The amount of force to apply towards angle specified in dir_angle_xz or dir_angle_y.
    * - dont_apply_immune
+     - Whether or not to apply immunity to the caster of the behavior.
    * - max damage
+     - The maximum amount of damage to be dealt.
    * - min damage
+     - The minimum amount of damage to be dealt.
    * - on_fail_armor
+     - The behavior to use on failure due to armor.
    * - on_fail_blocked
+     - The behavior to use on failure due to the attack being blocked.
    * - on_fail_immune
+     - The behavior to use on failure due to the target being immune.
    * - on_success
+     - The behavior to use on success of the attack.
    * - radius
+     - The radius of the attack.
    * - use_caster_velocity
+     - Whether or not to use the casters velocity.
    * - velocity_multiplier
+     - Value to multiply velocity by when attacking.
+
+DarkFlame Universe Parameter Notes
+----------------------------------
+
+As of February 11, 2022, only the following parameters are used:
+
+- max damage
+- min damage
+- on_success
+
+The following parameters are not attached to any behavior trees or skills from live:
```

Comment:
Huh, checking live is ususally how I collected the parameter list above. Need to check that again.

Reply:
There could be an error here with my SQL queries...  As far as I could tell though most behaviors that had these parameters were attached to skills with zero information besides the skillID and some filler info.  Since it is technically in the database Ill just mention instead that they are not attached to any usable skills, but they do exist as parameters for some or something along those lines.

Reply:
Have you checked scripts (or object config in level files) for these skills? Not being used in ObjectSkills doesn't have to mean they're unused

Reply:
Ah good point.  Ill check around to see if any of these are used.